### PR TITLE
[maven] Maven 3.6.3

### PR DESCRIPTION
**Pull request by carlossg**

### CODE REVIEW

File: library/maven [hunk 1]
```diff
@@ -1,64 +1,64 @@
...
 Architectures: amd64
-GitCommit: 081d9d041aa640694fceabc02c72fe93d5cd42cd
+GitCommit: d3dd6bc261c6173c5e52e3a7a36b6a3d8d2800b4
 Directory: jdk-8
 
-Tags: 3.6.2-ibmjava-8-alpine, 3.6.2-ibmjava-alpine, 3.6-ibmjava-8-alpine, 3.6-ibmjava-alpine, 3-ibmjava-8-alpine, ibmjava-alpine
+Tags: 3.6.3-ibmjava-8-alpine, 3.6.3-ibmjava-alpine, 3.6-ibmjava-8-alpine, 3.6-ibmjava-alpine, 3-ibmjava-8-alpine, ibmjava-alpine
 Architectures: amd64
-GitCommit: 081d9d041aa640694fceabc02c72fe93d5cd42cd
+GitCommit: d3dd6bc261c6173c5e52e3a7a36b6a3d8d2800b4
```
Comment (yosifkit) [inline library/maven]: This commit doesn't seem to exist on https://github.com/carlossg/docker-maven

Comment (carlossg) [reply library/maven]: 🤦‍♂  pushed now

**Comment by tianon** [comment]
<details>
<summary>Diff:</summary>

```diff
diff --git a/_bashbrew-list b/_bashbrew-list
index 4eb1956..fb8743c 100644
--- a/_bashbrew-list
+++ b/_bashbrew-list
@@ -28,22 +28,22 @@ maven:3.6-jdk-11-slim
 maven:3.6-jdk-13
 maven:3.6-jdk-14
 maven:3.6-slim
-maven:3.6.2
-maven:3.6.2-amazoncorretto-8
-maven:3.6.2-amazoncorretto-11
-maven:3.6.2-ibmjava
-maven:3.6.2-ibmjava-8
-maven:3.6.2-ibmjava-8-alpine
-maven:3.6.2-ibmjava-alpine
-maven:3.6.2-jdk-8
-maven:3.6.2-jdk-8-openj9
-maven:3.6.2-jdk-8-slim
-maven:3.6.2-jdk-11
-maven:3.6.2-jdk-11-openj9
-maven:3.6.2-jdk-11-slim
-maven:3.6.2-jdk-13
-maven:3.6.2-jdk-14
-maven:3.6.2-slim
+maven:3.6.3
+maven:3.6.3-amazoncorretto-8
+maven:3.6.3-amazoncorretto-11
+maven:3.6.3-ibmjava
+maven:3.6.3-ibmjava-8
+maven:3.6.3-ibmjava-8-alpine
+maven:3.6.3-ibmjava-alpine
+maven:3.6.3-jdk-8
+maven:3.6.3-jdk-8-openj9
+maven:3.6.3-jdk-8-slim
+maven:3.6.3-jdk-11
+maven:3.6.3-jdk-11-openj9
+maven:3.6.3-jdk-11-slim
+maven:3.6.3-jdk-13
+maven:3.6.3-jdk-14
+maven:3.6.3-slim
 maven:amazoncorretto
 maven:ibmjava-alpine
 maven:latest
diff --git a/maven_3-amazoncorretto-8/Dockerfile b/maven_3-amazoncorretto-8/Dockerfile
index 15e4368..bbe773e 100644
--- a/maven_3-amazoncorretto-8/Dockerfile
+++ b/maven_3-amazoncorretto-8/Dockerfile
@@ -1,8 +1,8 @@
 FROM amazoncorretto:8
 
-ARG MAVEN_VERSION=3.6.2
+ARG MAVEN_VERSION=3.6.3
 ARG USER_HOME_DIR="/root"
-ARG SHA=d941423d115cd021514bfd06c453658b1b3e39e6240969caf4315ab7119a77299713f14b620fb2571a264f8dff2473d8af3cb47b05acf0036fc2553199a5c1ee
+ARG SHA=c35a1803a6e70a126e80b2b3ae33eed961f83ed74d18fcd16909b2d44d7dada3203f1ffe726c17ef8dcca2dcaa9fca676987befeadc9b9f759967a8cb77181c0
 ARG BASE_URL=https://apache.osuosl.org/maven/maven-3/${MAVEN_VERSION}/binaries
 
 RUN yum install -y tar which gzip
diff --git a/maven_3-ibmjava-8/Dockerfile b/maven_3-ibmjava-8/Dockerfile
index 3a70e67..3737f27 100644
--- a/maven_3-ibmjava-8/Dockerfile
+++ b/maven_3-ibmjava-8/Dockerfile
@@ -1,8 +1,8 @@
 FROM ibmjava:8-sdk
 
-ARG MAVEN_VERSION=3.6.2
+ARG MAVEN_VERSION=3.6.3
 ARG USER_HOME_DIR="/root"
-ARG SHA=d941423d115cd021514bfd06c453658b1b3e39e6240969caf4315ab7119a77299713f14b620fb2571a264f8dff2473d8af3cb47b05acf0036fc2553199a5c1ee
+ARG SHA=c35a1803a6e70a126e80b2b3ae33eed961f83ed74d18fcd16909b2d44d7dada3203f1ffe726c17ef8dcca2dcaa9fca676987befeadc9b9f759967a8cb77181c0
 ARG BASE_URL=https://apache.osuosl.org/maven/maven-3/${MAVEN_VERSION}/binaries
 
 RUN apt-get update && apt-get install -y curl \
diff --git a/maven_3-jdk-11-openj9/Dockerfile b/maven_3-jdk-11-openj9/Dockerfile
index fa45210..556e148 100644
--- a/maven_3-jdk-11-openj9/Dockerfile
+++ b/maven_3-jdk-11-openj9/Dockerfile
@@ -1,8 +1,8 @@
 FROM adoptopenjdk:11-jdk-openj9-bionic
 
-ARG MAVEN_VERSION=3.6.2
+ARG MAVEN_VERSION=3.6.3
 ARG USER_HOME_DIR="/root"
-ARG SHA=d941423d115cd021514bfd06c453658b1b3e39e6240969caf4315ab7119a77299713f14b620fb2571a264f8dff2473d8af3cb47b05acf0036fc2553199a5c1ee
+ARG SHA=c35a1803a6e70a126e80b2b3ae33eed961f83ed74d18fcd16909b2d44d7dada3203f1ffe726c17ef8dcca2dcaa9fca676987befeadc9b9f759967a8cb77181c0
 ARG BASE_URL=https://apache.osuosl.org/maven/maven-3/${MAVEN_VERSION}/binaries
 
 RUN mkdir -p /usr/share/maven /usr/share/maven/ref \
diff --git a/maven_3-jdk-13/Dockerfile b/maven_3-jdk-13/Dockerfile
index 9e10c0d..3bc031e 100644
--- a/maven_3-jdk-13/Dockerfile
+++ b/maven_3-jdk-13/Dockerfile
@@ -1,8 +1,8 @@
 FROM openjdk:13-jdk
 
-ARG MAVEN_VERSION=3.6.2
+ARG MAVEN_VERSION=3.6.3
 ARG USER_HOME_DIR="/root"
-ARG SHA=d941423d115cd021514bfd06c453658b1b3e39e6240969caf4315ab7119a77299713f14b620fb2571a264f8dff2473d8af3cb47b05acf0036fc2553199a5c1ee
+ARG SHA=c35a1803a6e70a126e80b2b3ae33eed961f83ed74d18fcd16909b2d44d7dada3203f1ffe726c17ef8dcca2dcaa9fca676987befeadc9b9f759967a8cb77181c0
 ARG BASE_URL=https://apache.osuosl.org/maven/maven-3/${MAVEN_VERSION}/binaries
 
 RUN mkdir -p /usr/share/maven /usr/share/maven/ref \
diff --git a/maven_3-jdk-14/Dockerfile b/maven_3-jdk-14/Dockerfile
index 3bfcb7b..72be289 100644
--- a/maven_3-jdk-14/Dockerfile
+++ b/maven_3-jdk-14/Dockerfile
@@ -1,8 +1,8 @@
 FROM openjdk:14-jdk
 
-ARG MAVEN_VERSION=3.6.2
+ARG MAVEN_VERSION=3.6.3
 ARG USER_HOME_DIR="/root"
-ARG SHA=d941423d115cd021514bfd06c453658b1b3e39e6240969caf4315ab7119a77299713f14b620fb2571a264f8dff2473d8af3cb47b05acf0036fc2553199a5c1ee
+ARG SHA=c35a1803a6e70a126e80b2b3ae33eed961f83ed74d18fcd16909b2d44d7dada3203f1ffe726c17ef8dcca2dcaa9fca676987befeadc9b9f759967a8cb77181c0
 ARG BASE_URL=https://apache.osuosl.org/maven/maven-3/${MAVEN_VERSION}/binaries
 
 RUN mkdir -p /usr/share/maven /usr/share/maven/ref \
diff --git a/maven_3-jdk-8-openj9/Dockerfile b/maven_3-jdk-8-openj9/Dockerfile
index dee24e8..df7e1e9 100644
--- a/maven_3-jdk-8-openj9/Dockerfile
+++ b/maven_3-jdk-8-openj9/Dockerfile
@@ -1,8 +1,8 @@
 FROM adoptopenjdk:8-jdk-openj9-bionic
 
-ARG MAVEN_VERSION=3.6.2
+ARG MAVEN_VERSION=3.6.3
 ARG USER_HOME_DIR="/root"
-ARG SHA=d941423d115cd021514bfd06c453658b1b3e39e6240969caf4315ab7119a77299713f14b620fb2571a264f8dff2473d8af3cb47b05acf0036fc2553199a5c1ee
+ARG SHA=c35a1803a6e70a126e80b2b3ae33eed961f83ed74d18fcd16909b2d44d7dada3203f1ffe726c17ef8dcca2dcaa9fca676987befeadc9b9f759967a8cb77181c0
 ARG BASE_URL=https://apache.osuosl.org/maven/maven-3/${MAVEN_VERSION}/binaries
 
 RUN apt-get update && apt-get install -y curl \
diff --git a/maven_3-jdk-8-slim/Dockerfile b/maven_3-jdk-8-slim/Dockerfile
index b6bdb09..47dfe2e 100644
--- a/maven_3-jdk-8-slim/Dockerfile
+++ b/maven_3-jdk-8-slim/Dockerfile
@@ -1,8 +1,8 @@
 FROM openjdk:8-jdk-slim
 
-ARG MAVEN_VERSION=3.6.2
+ARG MAVEN_VERSION=3.6.3
 ARG USER_HOME_DIR="/root"
-ARG SHA=d941423d115cd021514bfd06c453658b1b3e39e6240969caf4315ab7119a77299713f14b620fb2571a264f8dff2473d8af3cb47b05acf0036fc2553199a5c1ee
+ARG SHA=c35a1803a6e70a126e80b2b3ae33eed961f83ed74d18fcd16909b2d44d7dada3203f1ffe726c17ef8dcca2dcaa9fca676987befeadc9b9f759967a8cb77181c0
 ARG BASE_URL=https://apache.osuosl.org/maven/maven-3/${MAVEN_VERSION}/binaries
 
 RUN apt-get update && \
diff --git a/maven_3-jdk-8/Dockerfile b/maven_3-jdk-8/Dockerfile
index 1b70c51..566ffff 100644
--- a/maven_3-jdk-8/Dockerfile
+++ b/maven_3-jdk-8/Dockerfile
@@ -1,8 +1,8 @@
 FROM openjdk:8-jdk
 
-ARG MAVEN_VERSION=3.6.2
+ARG MAVEN_VERSION=3.6.3
 ARG USER_HOME_DIR="/root"
-ARG SHA=d941423d115cd021514bfd06c453658b1b3e39e6240969caf4315ab7119a77299713f14b620fb2571a264f8dff2473d8af3cb47b05acf0036fc2553199a5c1ee
+ARG SHA=c35a1803a6e70a126e80b2b3ae33eed961f83ed74d18fcd16909b2d44d7dada3203f1ffe726c17ef8dcca2dcaa9fca676987befeadc9b9f759967a8cb77181c0
 ARG BASE_URL=https://apache.osuosl.org/maven/maven-3/${MAVEN_VERSION}/binaries
 
 RUN mkdir -p /usr/share/maven /usr/share/maven/ref \
diff --git a/maven_amazoncorretto/Dockerfile b/maven_amazoncorretto/Dockerfile
index 577b4b8..d04fbc1 100644
--- a/maven_amazoncorretto/Dockerfile
+++ b/maven_amazoncorretto/Dockerfile
@@ -1,8 +1,8 @@
 FROM amazoncorretto:11
 
-ARG MAVEN_VERSION=3.6.2
+ARG MAVEN_VERSION=3.6.3
 ARG USER_HOME_DIR="/root"
-ARG SHA=d941423d115cd021514bfd06c453658b1b3e39e6240969caf4315ab7119a77299713f14b620fb2571a264f8dff2473d8af3cb47b05acf0036fc2553199a5c1ee
+ARG SHA=c35a1803a6e70a126e80b2b3ae33eed961f83ed74d18fcd16909b2d44d7dada3203f1ffe726c17ef8dcca2dcaa9fca676987befeadc9b9f759967a8cb77181c0
 ARG BASE_URL=https://apache.osuosl.org/maven/maven-3/${MAVEN_VERSION}/binaries
 
 RUN yum install -y tar which gzip
diff --git a/maven_ibmjava-alpine/Dockerfile b/maven_ibmjava-alpine/Dockerfile
index 1f2a6f5..0367e0e 100644
--- a/maven_ibmjava-alpine/Dockerfile
+++ b/maven_ibmjava-alpine/Dockerfile
@@ -2,9 +2,9 @@ FROM ibmjava:8-sdk-alpine
 
 RUN apk add --no-cache curl tar bash procps
 
-ARG MAVEN_VERSION=3.6.2
+ARG MAVEN_VERSION=3.6.3
 ARG USER_HOME_DIR="/root"
-ARG SHA=d941423d115cd021514bfd06c453658b1b3e39e6240969caf4315ab7119a77299713f14b620fb2571a264f8dff2473d8af3cb47b05acf0036fc2553199a5c1ee
+ARG SHA=c35a1803a6e70a126e80b2b3ae33eed961f83ed74d18fcd16909b2d44d7dada3203f1ffe726c17ef8dcca2dcaa9fca676987befeadc9b9f759967a8cb77181c0
 ARG BASE_URL=https://apache.osuosl.org/maven/maven-3/${MAVEN_VERSION}/binaries
 
 RUN mkdir -p /usr/share/maven /usr/share/maven/ref \
diff --git a/maven_latest/Dockerfile b/maven_latest/Dockerfile
index 9efd4dc..a15e045 100644
--- a/maven_latest/Dockerfile
+++ b/maven_latest/Dockerfile
@@ -1,8 +1,8 @@
 FROM openjdk:11-jdk
 
-ARG MAVEN_VERSION=3.6.2
+ARG MAVEN_VERSION=3.6.3
 ARG USER_HOME_DIR="/root"
-ARG SHA=d941423d115cd021514bfd06c453658b1b3e39e6240969caf4315ab7119a77299713f14b620fb2571a264f8dff2473d8af3cb47b05acf0036fc2553199a5c1ee
+ARG SHA=c35a1803a6e70a126e80b2b3ae33eed961f83ed74d18fcd16909b2d44d7dada3203f1ffe726c17ef8dcca2dcaa9fca676987befeadc9b9f759967a8cb77181c0
 ARG BASE_URL=https://apache.osuosl.org/maven/maven-3/${MAVEN_VERSION}/binaries
 
 RUN mkdir -p /usr/share/maven /usr/share/maven/ref \
diff --git a/maven_slim/Dockerfile b/maven_slim/Dockerfile
index 4a79b07..f0334d7 100644
--- a/maven_slim/Dockerfile
+++ b/maven_slim/Dockerfile
@@ -1,8 +1,8 @@
 FROM openjdk:11-jdk-slim
 
-ARG MAVEN_VERSION=3.6.2
+ARG MAVEN_VERSION=3.6.3
 ARG USER_HOME_DIR="/root"
-ARG SHA=d941423d115cd021514bfd06c453658b1b3e39e6240969caf4315ab7119a77299713f14b620fb2571a264f8dff2473d8af3cb47b05acf0036fc2553199a5c1ee
+ARG SHA=c35a1803a6e70a126e80b2b3ae33eed961f83ed74d18fcd16909b2d44d7dada3203f1ffe726c17ef8dcca2dcaa9fca676987befeadc9b9f759967a8cb77181c0
 ARG BASE_URL=https://apache.osuosl.org/maven/maven-3/${MAVEN_VERSION}/binaries
 
 RUN apt-get update && \
```

</details>

**Comment by tianon** [comment]
LGTM

Build test of #7037; 54800331; `amd64` (`maven`):

```console

$ bashbrew build maven:3.6.3-jdk-11-openj9
Pulling adoptopenjdk:11-jdk-openj9-bionic (maven:3.6.3-jdk-11-openj9)
Building bashbrew/cache:bff0c2b91ba033803ecd8786c04b7248ea3a32e6f7959bf8c73790d5e3441123 (maven:3.6.3-jdk-11-openj9)
Tagging maven:3.6.3-jdk-11-openj9
Tagging maven:3.6-jdk-11-openj9
Tagging maven:3-jdk-11-openj9

$ test/run.sh maven:3.6.3-jdk-11-openj9
testing maven:3.6.3-jdk-11-openj9
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...passed
	'override-cmd' [4/4]...passed


$ bashbrew build maven:3.6.3-jdk-11-slim
Pulling openjdk:11-jdk-slim (maven:3.6.3-jdk-11-slim)
Building bashbrew/cache:e71fb4ecba3a0bc9a74214fbda87cab0c1a0a09f71c4a5de1235851f159e74c1 (maven:3.6.3-jdk-11-slim)
Tagging maven:3.6.3-jdk-11-slim
Tagging maven:3.6.3-slim
Tagging maven:3.6-jdk-11-slim
Tagging maven:3.6-slim
Tagging maven:3-jdk-11-slim
Tagging maven:slim

$ test/run.sh maven:3.6.3-jdk-11-slim
testing maven:3.6.3-jdk-11-slim
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...passed
	'override-cmd' [4/4]...passed


$ bashbrew build maven:3.6.3-jdk-11
Pulling openjdk:11-jdk (maven:3.6.3-jdk-11)
Building bashbrew/cache:c82dbff1716811357b3d9bed7d2dad12b6d0490bd1d217250115fc40eeb77165 (maven:3.6.3-jdk-11)
Tagging maven:3.6.3-jdk-11
Tagging maven:3.6.3
Tagging maven:3.6-jdk-11
Tagging maven:3.6
Tagging maven:3-jdk-11
Tagging maven:3
Tagging maven:latest

$ test/run.sh maven:3.6.3-jdk-11
testing maven:3.6.3-jdk-11
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...passed
	'override-cmd' [4/4]...passed


$ bashbrew build maven:3.6.3-jdk-13
Pulling openjdk:13-jdk (maven:3.6.3-jdk-13)
Building bashbrew/cache:c43757105a329bff935ce9585a96c2cd801304ea1638778a7299cdb98a48909d (maven:3.6.3-jdk-13)
Tagging maven:3.6.3-jdk-13
Tagging maven:3.6-jdk-13
Tagging maven:3-jdk-13

$ test/run.sh maven:3.6.3-jdk-13
testing maven:3.6.3-jdk-13
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...passed
	'override-cmd' [4/4]...passed


$ bashbrew build maven:3.6.3-jdk-14
Pulling openjdk:14-jdk (maven:3.6.3-jdk-14)
Building bashbrew/cache:1d5c09e6814e3790c67cfcd5dec11abf52dd464948fc7f206627410d451aeaad (maven:3.6.3-jdk-14)
Tagging maven:3.6.3-jdk-14
Tagging maven:3.6-jdk-14
Tagging maven:3-jdk-14

$ test/run.sh maven:3.6.3-jdk-14
testing maven:3.6.3-jdk-14
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...passed
	'override-cmd' [4/4]...passed


$ bashbrew build maven:3.6.3-jdk-8-openj9
Pulling adoptopenjdk:8-jdk-openj9-bionic (maven:3.6.3-jdk-8-openj9)
Building bashbrew/cache:5f69bf295f7d0d91fee4406cfef564ab9175d39e7261e2eb506abb25871bdab7 (maven:3.6.3-jdk-8-openj9)
Tagging maven:3.6.3-jdk-8-openj9
Tagging maven:3.6-jdk-8-openj9
Tagging maven:3-jdk-8-openj9

$ test/run.sh maven:3.6.3-jdk-8-openj9
testing maven:3.6.3-jdk-8-openj9
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...passed
	'override-cmd' [4/4]...passed


$ bashbrew build maven:3.6.3-jdk-8-slim
Pulling openjdk:8-jdk-slim (maven:3.6.3-jdk-8-slim)
Building bashbrew/cache:764f24e889a654622862259b896558d6e255257d96d8afd3908ba9c0b378c802 (maven:3.6.3-jdk-8-slim)
Tagging maven:3.6.3-jdk-8-slim
Tagging maven:3.6-jdk-8-slim
Tagging maven:3-jdk-8-slim

$ test/run.sh maven:3.6.3-jdk-8-slim
testing maven:3.6.3-jdk-8-slim
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...passed
	'override-cmd' [4/4]...passed


$ bashbrew build maven:3.6.3-jdk-8
Pulling openjdk:8-jdk (maven:3.6.3-jdk-8)
Building bashbrew/cache:9edbc15eade2bc216f182cecebbefc5018432d44c559d9e481a44b8388d7c3a5 (maven:3.6.3-jdk-8)
Tagging maven:3.6.3-jdk-8
Tagging maven:3.6-jdk-8
Tagging maven:3-jdk-8

$ test/run.sh maven:3.6.3-jdk-8
testing maven:3.6.3-jdk-8
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...passed
	'override-cmd' [4/4]...passed


$ bashbrew build maven:3.6.3-ibmjava-8-alpine
Pulling ibmjava:8-sdk-alpine (maven:3.6.3-ibmjava-8-alpine)
Building bashbrew/cache:83148adf48428f87bebaf230381db7a25b4c04f8c5757b6fee81e6efbc53acec (maven:3.6.3-ibmjava-8-alpine)
Tagging maven:3.6.3-ibmjava-8-alpine
Tagging maven:3.6.3-ibmjava-alpine
Tagging maven:3.6-ibmjava-8-alpine
Tagging maven:3.6-ibmjava-alpine
Tagging maven:3-ibmjava-8-alpine
Tagging maven:ibmjava-alpine

$ test/run.sh maven:3.6.3-ibmjava-8-alpine
testing maven:3.6.3-ibmjava-8-alpine
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...passed
	'override-cmd' [4/4]...passed


$ bashbrew build maven:3.6.3-ibmjava-8
Building bashbrew/cache:74e16c5a6888bdb2cb6e4b6117fd25b1fd58a9aa4fea58d143033f9b3d1302c0 (maven:3.6.3-ibmjava-8)
Tagging maven:3.6.3-ibmjava-8
Tagging maven:3.6.3-ibmjava
Tagging maven:3.6-ibmjava-8
Tagging maven:3.6-ibmjava
Tagging maven:3-ibmjava-8

$ test/run.sh maven:3.6.3-ibmjava-8
testing maven:3.6.3-ibmjava-8
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...passed
	'override-cmd' [4/4]...passed


$ bashbrew build maven:3.6.3-amazoncorretto-11
Pulling amazoncorretto:11 (maven:3.6.3-amazoncorretto-11)
Building bashbrew/cache:f86717812c967468d28d03373bf38b857cb420ba392ef6eef0cb11f4d214e860 (maven:3.6.3-amazoncorretto-11)
Tagging maven:3.6.3-amazoncorretto-11
Tagging maven:3.6-amazoncorretto-11
Tagging maven:3-amazoncorretto-11
Tagging maven:3-amazoncorretto
Tagging maven:amazoncorretto

$ test/run.sh maven:3.6.3-amazoncorretto-11
testing maven:3.6.3-amazoncorretto-11
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...warning: garbage password detected for 'root': '*LOCK*'
passed
	'override-cmd' [4/4]...passed


$ bashbrew build maven:3.6.3-amazoncorretto-8
Pulling amazoncorretto:8 (maven:3.6.3-amazoncorretto-8)
Building bashbrew/cache:6c5fb4840119ab32c8f87bad08883f6c8dd360de2ddd03d71508f5eb815fddd0 (maven:3.6.3-amazoncorretto-8)
Tagging maven:3.6.3-amazoncorretto-8
Tagging maven:3.6-amazoncorretto-8
Tagging maven:3-amazoncorretto-8

$ test/run.sh maven:3.6.3-amazoncorretto-8
testing maven:3.6.3-amazoncorretto-8
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...warning: garbage password detected for 'root': '*LOCK*'
passed
	'override-cmd' [4/4]...passed

```

**Pull request merged**